client/src/canvas/ViewSidebar.res

```diff
@@ -54,97 +58,66 @@ and category = {
   name: string,
   plusButton: option<msg>,
   iconAction: option<msg>,
-  classname: string,
+  icon: Html.html<msg>,
+  emptyName: string, // what if there's none
```

Comment:
```suggestion
  emptyName: string, // if none are present, used in `"No " ++ emptyname` (e.g. "No datastores")
```
I found `emptyName` unobvious - had to look at the usages, this would have helped